aws/data_source_aws_instance.go

```diff
@@ -51,6 +51,10 @@ func dataSourceAwsInstance() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"host_id": {
```

Comment:
Can you please add documentation for this new attribute in `website/docs/d/instance.html.markdown`? Thanks

Reply:
added documentation.